src/commands/CommandUtility.ts

```diff
@@ -107,4 +113,23 @@ export class CommandUtility implements ICommandUtility {
 			}
 		}
 	}
+
+	private async handleQuickCreate(handler: Handler, args: string[]): Promise<void> {
+		if (!args || args.length === 0) {
+		  await handler.sendDefault();
+		  return;
+		}
+	
+		switch (args[0].toLowerCase()) {
+		  case CommandParam.CREATE: {
+			await handler.CreateQuickReply(args);
+			break;
+		  }
+		  default: {
```

Comment:
this is the third place where you please .sendDefault() is there  any other way to achieve check without repeating this